packages/core/client/src/schema-component/antd/input-number/InputNumber.tsx

```diff
@@ -4,12 +4,12 @@ import React from 'react';
 import { ReadPretty } from './ReadPretty';
 
 export const InputNumber = connect((props) => {
-  const { onChange, ...others } = props;
+  const { onChange, step, ...others } = props;
   const field = useField<any>();
   const handleChange = (v) => {
     onChange(parseFloat(v));
   };
-  return <AntdNumber onChange={handleChange} {...others}  />;
+  return <AntdNumber step={Number(step) || 1} onChange={handleChange} {...others}  />;
```

Comment:
```suggestion
  return <AntdNumber step={step} onChange={handleChange} {...others}  />;
```
默认值放上面，这样就可以去掉 `||` 操作符了，另外 `step` 是支持 `string` 的，所以不需要转成 `number`
![image](https://user-images.githubusercontent.com/38434641/232362841-619d9154-10f8-41c1-be38-556393f205d3.png)
参考文档：https://4x.ant.design/components/input-number-cn/

Reply:
重新赋值的原因是因为有时候拿到的 step 值是一个字符串 0，所以才会造成点击步进器没有反应，所以做了这样的处理

Reply:
这个 commit 的 cla 有点问题，review之后我重新 pr

Reply:
> 重新赋值的原因是因为有时候拿到的 step 值是一个字符串 0，所以才会造成点击步进器没有反应，所以做了这样的处理

感觉应该要排查下为什么会有 0 传过来，是不是有特别的用意？

Reply:
👌